runtime/dart_lifecycle_unittests.cc

```diff
@@ -1,13 +1,15 @@
 // Copyright 2013 The Flutter Authors. All rights reserved.
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
+// FLUTTER_NOLINT
```

Comment:
We're supposed to be trying to get rid of these =)

Reply:
Oh, or maybe that's why you hit the linter bugs...

Reply:
Yeah, since I touched the file, the linter linted the whole file. Didn't think reworking the APIs to make the linter happy was in scope for this patch.